packages/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -88,7 +89,7 @@ export const CardHeader: React.FunctionComponent<CardHeaderProps> = ({
   isToggleRightAligned,
   ...props
 }: CardHeaderProps) => {
-  const uniqueId = React.useId();
+  const uniqueId = lodashId();
```

Comment:
We are avoid using lodash in our `react-core` We actually had a tech debt issu to remove it entirely from the library. Could you please use our utility functions instead.

https://github.com/patternfly/patternfly-react/blob/main/packages/react-core/src/helpers/GenerateId/GenerateId.ts